src/fake.ts

```diff
@@ -88,11 +89,11 @@ export class Fake {
     const parts = method.split('.');
 
     if (this.faker[parts[0]] == null) {
-      throw new Error('Invalid module: ' + parts[0]);
+      throw new FakerError('Invalid module: ' + parts[0]);
```

Comment:
Why not use string templates like we do in other parts of the code??

```suggestion
      throw new FakerError(`Invalid module: ${parts[0]}`);
```

Reply:
This applies in many places...

Reply:
We don't have the [prefer-template linter rule](https://eslint.org/docs/rules/prefer-template) so I don't see why this would be a required change. We do "normal" string concatenation all over the project.

Reply:
Indeed. I will propose this change in a separate PR as we currently use it in some places already...

Reply:
Or should we rework this when you propose that change?

Reply:
FYI https://github.com/faker-js/faker/pull/732 - easily fixable with `eslint --fix .` so no worries, we can move forward with this PR.